app/routes.js

```diff
@@ -60,7 +61,10 @@ var BreadcrumbMaker = require('../lib/js/breadcrumb_maker.js');
         res.render('content', { content: content, breadcrumb: breadcrumb, taxons: taxons, whitehall: whitehall, homepage_url: '/'});
       },
       function (e) {
-        res.render('content', {content: 'Page not found', homepage_url: '/'});
+        readFile(error404Page).then(function(data) {
+          content = data.toString();
```

Comment:
Ah cool, hadn't though of doing that instead! As a plus, it also fixes the whitespace issue.